Move QuokkADB from PlatformIO to CMake

morio opened the pull request:
The wizio-pico platform that QuokkADB used in PlatformIO is dead. Switching to CMake for latest version of the pico-sdk v1.5.0

morio committed:
Build QuokkADB with cmake for new pico sdk

Wizio on platformio doesn't support the latest pico SDK.
Building with cmake allows the project to use the new
SDK version.

morio committed:
Removed vtable errors

morio committed:
Merge pull request #8 from rabbitholecomputing/cmake

QuokkADB switching from PlatformIO to Cmake

akuker commented on src/firmware/lib/adb/src/adb.cpp:
```diff
@@ -491,9 +491,12 @@ void AdbInterface::ProcessCommand(int16_t cmd)
-          kbskiptimer = millis();
+// @DEBUG switch back after getting cmake to compile QuokkADB
+//          kbskiptimer = millis();
+            kbskiptimer = (time_us_64() / 1000);
```
@morio - did you intend for this change to remain? The comment looks like it might have been a temporary debug thing.

morio replied:
No, I'll fix it and do some more testing then commit to this PR.

morio replied:
@akuker the changes have been made.

morio committed:
Fix and remove DEBUG committed

Fixed and removed code with the DEBUG label.
Removed code that was commented out.
Added some hints on how to debug using vscode and a picoprobe.
Added compile date and time to firmware string.
Removed QuokkADB compile option from platformio.ini as compiling
has been moved the CMake.

morio committed:
Merge pull request #9 from rabbitholecomputing/cmake

Fix and remove DEBUG code
